src/poliastro/czml/utils.py

```diff
@@ -0,0 +1,127 @@
+from math import sqrt
+
+
+def tangential_plane_ellipsoid(x, y, z, a, b, c):
+    """
+    Given a point on the ellipsoid, find its tangential plane.
+
+    Given the ellipsoid defined by x^2/a^2 + y^2/b^2 + z^2/c^2 = 0 and
+    x0, y0, z0 the given point on the ellipse the tangential plane is:
+    2(x - x0)/a^2 + 2(y - y0)/b^2 + 2(z - z0)/c^2 = 0 =>
+    (2/a^2)x + (2/b^2)y + (2/c^2)z - 2(x0/a^2 + y0/b^2 + z0/c^2)
+
+    Parameters
+    ----------
+    x, y, z: int
+        Coordinates on the ellipse
+    a, b, c: int
+        Ellipsoid axises
+
+    Returns
+    -------
+    Returns the coefficients of the general form of the plane
+    Ax + By + Cz + D = 0
+    """
+    return [
+        2 / a ** 2,
+        2 / b ** 2,
+        2 / c ** 2,
+        -2 * (x / a ** 2 + y / b ** 2 + z / c ** 2),
+    ]
+
+
+def intersection_ellipsoid_line(a, b, c, u1, u2, u3, x, y, z):
+    """
+    Intersection of an ellipsoid defined by its axises a, b, c with the
+    line p + λu
+
+    Parameters
+    ----------
+    a, b, c: float
+        the ellipsoidal axises
+    u1, u2, u3: float
+        the line vector
+    x, y, z: float
+        a point of the line
+
+    Returns
+    -------
+    This returns both of the points intersecting the ellipsoid.
+
+    """
+    # Get rid of one parameter by translating the line's direction vector
+    k, m = u2 / u1, u3 / u1
+    t0 = -(
+        pow(a, 2) * pow(b, 2) * m * z
```

Comment:
Just one comment: any reason not to use `a ** 2`?

Reply:
Mainly for formatting reasons. I tried ``a ** 2`` and ``a * a``, but since the arithmetic expression is way too long to be considered concise, I thought that using ``pow`` is easier on the eye. That being said, that was before I reformatted with ``black`` which makes everything far more readable, so I can change it back if you think it fits in better with the rest.

Reply:
Yes, let's do `a ** 2` instead!